patches/minecraft/net/minecraft/world/item/Item.java.patch

```diff
@@ -167,6 +168,9 @@
 +   public Object getRenderPropertiesInternal() {
 +      return renderProperties;
 +   }
++   public Object getItemDecoratorHandler() {
```

Comment:
Now that we have the JetBrains annotations, this should probably be annotated with `@ApiStatus.Internal`.

Reply:
If he applies my suggestion it will be nuked and moved anyway.
i would like this in an external manager not in the item.